doc/manual.asciidoc

```diff
@@ -855,6 +855,12 @@ keys.
    stored as `.ninja_deps` in the `builddir`, see <<ref_toplevel,the
    discussion of `builddir`>>.
 
+`dynout`:: path to an optional _dynout file_ that contains the list
+  of outputs generated by the rule. The dynout file syntax except one
+  path per line. This is to make Ninja aware of dynamic outputs, so 
```

Comment:
I assume this is `expeccts one path per line`? Are there any escaping mechanisms? I guess this means that paths with embedded newlines are not supported (not that I expect they are anywhere else either, but it'd be good to know before reviewing the parsing code)?